node_modules/oae-admin/tenants/js/tenants.js

```diff
@@ -404,18 +385,36 @@ define(['jquery', 'oae.core', 'jquery.jeditable'], function($, oae) {
             $rootel.on('click', '.admin-table-striped-toggle', toggleContainer);
             // Stop a tenant
             $rootel.on('click', '.tenants-stop', stopTenantHandler);
-            // Stop all tenants
-            $rootel.on('click', '.tenants-stop-all', stopAllTenantsHandler);
             // Start a tenant
             $rootel.on('click', '.tenants-start', startTenantHandler);
-            // Start all tenants
-            $rootel.on('click', '.tenants-start-all', startAllTenantsHandler);
             // Log onto a tenant
             $rootel.on('click', '.tenants-login', loginOnTenantHandler);
+            // Listen to History.js state changes
+            $(window).on('statechange', renderTenants);
+        };
+
+        /**
+         * Render a single tenant row
+         *
+         * @param  {Tenant}     tenant     The tenant to render
+         */
+        var renderTenant = function(tenant) {
+            var $target = $('#tenants-row-' + oae.api.util.security().encodeForHTMLAttribute(tenant.alias), $rootel);
```

Comment:
This doesn't appear to be working, and it would be better to use the data attribute anyway